cmd/kpromo/cmd/pr/pr.go

```diff
@@ -140,10 +143,20 @@ func init() {
 		&promoteOpts.images,
 		"image",
 		"",
+		// TODO: Parameterize
 		[]string{""}, // default to a single empty string image filter which will promote all images matching the tag.
 		"image to promote. If not specified, all images matching the tag will be promoted",
 	)
 
+	PRCmd.PersistentFlags().StringSliceVarP(
+		&promoteOpts.images,
```

Comment:
should be &promoteOpts.digests?

Reply:
Eeep, good catch!

Reply:
Fixed in https://github.com/kubernetes-sigs/promo-tools/compare/846a0134b1f53b70ace027810b91cbbf4e739a67..a8b54d25f8f76249b2c0d942ae7513469cccc560.